gym-game2048/gym_game2048/envs/game_2048.py

```diff
@@ -1,30 +1,10 @@
 import numpy as np
-import math
-from numba import jitclass, njit, gdb_init
-from numba import uint32, int32, int32, b1, float64, float32
-
-spec = [
-    ("__board_size", int32),
-    ("__total_score", int32),
-    ("__score", int32),
-    ("__temp_board", uint32[:, :]),
-    ("__board", uint32[:, :]),
-    ("__done_merge", b1),
-    ("__done_cover_up", b1),
-    ("__invalid_count", uint32),
-    ("__total_count", uint32),
-    ("__invalid_move_warmup", uint32),
-    ("__invalid_move_threshold", float64),
-    ("__power_mat", uint32[:, :, :]),
-    ("__penalty", int32),
-]
-
-
-@jitclass(spec)
+
+
 class Game2048:
-    def __init__(self, board_size: int, invalid_move_warmup=16, invalid_move_threshold=0.1, penalty=-512):
+    def __init__(self, board_size: int, invalid_move_warmup=16, invalid_move_threshold=0.1, penalty=-512, dynamic_obstacle_interval=5, dynamic_obstacle_value=1):
```

Comment:
I made the `dynamic_obstacle_value` a variable that can be passed during initialization for better code readability. Let me know if you think this is overkill or if you'd prefer a different approach